cmake/onnxruntime_mlas.cmake

```diff
@@ -93,6 +93,35 @@ else()
       ${ONNXRUNTIME_ROOT}/core/mlas/lib/aarch64/sgemma.s
     )
 
+  elseif (CMAKE_SYSTEM_PROCESSOR MATCHES "^(i.86|x86?)$")
+
+    enable_language(ASM)
+
+    set(mlas_platform_srcs_sse2
+      ${ONNXRUNTIME_ROOT}/core/mlas/lib/x86/SgemmKernelSse2.S
+    )
+
+    set(mlas_platform_srcs_avx
+      ${ONNXRUNTIME_ROOT}/core/mlas/lib/x86/SgemmKernelAvx.S
+    )
+
+    set_source_files_properties(${mlas_common_srcs} PROPERTIES COMPILE_FLAGS "-msse2 -mavx")
```

Comment:
What if the machine doesn't support AVX?

Reply:
Agreed, my original change did not specify any of these switches for the common files. All files in the top level file should build with -msse2 or Eigen won’t use intrinsics either and that is bad.

Reply:
Removing the line.